pontoon/sync/management/commands/sync_projects.py

```diff
@@ -0,0 +1,49 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from pontoon.base.models import Project
+from pontoon.sync.tasks import sync_project
+
+
+class Command(BaseCommand):
+    args = '<project_slug project_slug ...>'
+    help = 'Synchronize database and remote repositories.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--no-commit',
+            action='store_true',
+            dest='no_commit',
+            default=False,
+            help='Do not commit changes to VCS'
+        )
+
+        parser.add_argument(
+            '--no-pull',
+            action='store_true',
+            dest='no_pull',
+            default=False,
+            help='Do not pull new commits from VCS'
+        )
+
+    def handle(self, *args, **options):
+        """
+        Collect the projects we want to sync and trigger worker jobs to
+        sync each one.
+        """
+        projects = Project.objects.filter(disabled=False)
+        if args:
+            projects = projects.filter(slug__in=args)
+
+        if len(projects) < 1:
```

Comment:
Second thing: maybe it rather should be projects.count() == len(args)? Because if someone calls this command with mispell, there's no other way to notice it.